mappings/1.1#12w03a.tinydiff

```diff
@@ -16,7 +16,7 @@ c	net/minecraft/unmapped/C_1013763
 	m	(F)V	m_3300694		m_3300694
 		p	1			p_1
 	m	()F	m_6481881		m_6481881
-	m	(Lnet/minecraft/unmapped/C_6385858;)Z	m_9004216		attack
+	m	(Lnet/minecraft/unmapped/C_6385858;)Z	m_9004216		tryAttack
```

Comment:
Having looked at the relevant methods, I think it'd make more sense to change the one in `LivingEntity`. `PlayerEntity.attack` handles things like hunger and item damage, in addition to dealing damage to the target entity. `LivingEntity.attack` only handles dealing damage. So I'd rename that one to something like `doDamage` or `dealDamage`. What are your thoughts?

Reply:
i've used `tryAttack`, as i explained on discord, because it can fail, plus its what yarn calls it on 1.14+.

this said, `doDamage` and `dealDamage` while might be more descriptive, sound a bit more confusing, as one expects an entity to attack, not to just "do damage"; but if you think its better, i can rename it to either of the two, don't want to stall this for an opinion.

Reply:
would `tryDamage` work?